.changeset/moody-spies-learn.md

```diff
@@ -0,0 +1,19 @@
+---
+"@astrojs/markdoc": patch
+---
+
+Add automatic resolution for Markdoc partials. This allows you to render other Markdoc files inside of a given entry. Reference files using the `partial` tag with a `file` attribute for the relative file path:
+
+```md
+<!--src/content/blog/post.mdoc-->
+
+{% partial file="my-partials/_diagram.mdoc" /%}
```

Comment:
It's worth mentioning here or in the docs from **which folder** the partials are solved from. `srcDir`?

Reply:
Ah, it is a relative path from whatever file you're currently in. So no folder specifically! This is clearer in the docs, so I'll update the changeset to match